### ISSUE TITLE
INFO messages about performingChange method

### ISSUE THREAD

**Issue by DiamondJim87** [das-developers/das2java#67]
Not sure when this started, but in my console as our tools run the following kinds of messages frequently are printed now.

<pre>
INFO: no lock object found for tcaload_axis_29
Jul 25, 2023 9:51:59 AM org.das2.system.ChangesSupport changePerformed
INFO: expect value for changeCount tcaload_axis_29, was performingChange called?
</pre>

I'm not aware of a code change on our end that caused this. Any idea what these mean or what I could change so as not to trigger them?

**Comment by jbfaden** [comment]
I can't think of any changes made in the past 5 years with the tca loading.  I take it everything is working properly.  I think we should just lower the message from INFO to FINE if things are working.  

Honestly I'd have to study it all to remind myself what the concern was.

**Comment by DiamondJim87** [comment]
After I filed this, I did look in other issues, and #32 seemed possibly related. That was about a year ago. Is it possible that change is interacting with the TCA code?

**Comment by jbfaden** [comment]
Whoops, I didn't mean to close this.

**Comment by jbfaden** [comment]
Does this continue to be an issue?

**Comment by DiamondJim87** [comment]
Yes I just confirmed it still happens. As the message suggests, it only happens when I have TCA functions.
